app/views/curation_concerns/base/_member.html.erb

```diff
@@ -0,0 +1,16 @@
+<tr class="<%= dom_class(member) %> attributes">
+  <td class="thumbnail">
+    <% if member.reader? %>
```

Comment:
Shouldn't the download controller just show the appropriate (default if no read access) thumbnail?